engine/src/cmd/collide2/opcodetypes.h

```diff
@@ -67,46 +67,14 @@
  * types. Use the explicitly-sized types sparingly.
  * @{ */
 
-#if !defined(HAVE_STDINT_H) && !defined(HAVE_INTTYPES_H)
-/// unsigned 8-bit integer (0..255)
-typedef unsigned char uint8;
-/// signed 8-bit integer (-128..127)
-typedef char int8;
-/// unsigned 16-bit integer (0..65 535)
-typedef unsigned short uint16;
-/// signed 16-bit integer (-32 768..32 767)
-typedef short int16;
-/// unsigned 32-bit integer (0..4 294 967 295)
-typedef unsigned int uint32;
-/// signed 32-bit integer (-2 147 483 648..2 147 483 647)
-typedef int int32;
-#if defined(CS_COMPILER_GCC)
-#ifndef __STRICT_ANSI__
-/// unsigned 64-bit integer
-typedef unsigned long long uint64;
-/// signed 64-bit integer
-typedef long long int64;
-#endif
-#elif defined(CS_COMPILER_MSVC) || defined(CS_COMPILER_BCC)
-/// unsigned 64 bit integer
-typedef unsigned __int64 uint64;
-/// signed 64 bit integer
-typedef __int64 int64;
-#else
-#error Do not know how to declare 64-bit integers
-#endif // CS_COMPILER_GCC
-
-#else // CS_HAVE_STDINT_H || CS_HAVE_INTTYPES_H
-
-typedef uint8_t uint8;
-typedef int8_t int8;
-typedef uint16_t uint16;
-typedef int16_t int16;
-typedef uint32_t uint32;
-typedef int32_t int32;
-typedef uint64_t uint64;
-typedef int64_t int64;
-#endif
+typedef uint8_t opc_uint8;
+typedef int8_t opc_int8;
+typedef uint16_t opc_uint16;
+typedef int16_t opc_int16;
+typedef uint32_t opc_uint32;
+typedef int32_t opc_int32;
+typedef uint64_t opc_uint64;
+typedef int64_t opc_int64;
```

Comment:
Another key change. Let's just use the C++11 standard fixed-width integer types, regardless of platform. It should work.